src/golang/lib/livy/entrypoints.go

```diff
@@ -0,0 +1,65 @@
+package livy
```

Comment:
Do you see a future refactor where we might put this python code in our python codebase so it's easier to maintain?

Reply:
Yep, I have to investigate the best way to do this. We want to ship this code as a string and need it available from the server binary. Not sure if the server binary has access to aqueduct-ml package.